src/test/HostActivationTests/GivenThatICareAboutLightupAppActivation.cs

```diff
@@ -128,6 +128,61 @@ public void Muxer_activation_of_LightupApp_WithLightupLib_Succeeds()
                 .HaveStdOutContaining("Hello LightupClient");
         }
 
+        // Attempt to run the app with a bad lightup deps.json specified.
```

Comment:
Test for roll forward fix?

Reply:
Yeah agree we should have one. I'll see if this is feasible